Snippets/Sqs/Sqs_All/QueueCreation/QueueCreation.ps1

```diff
@@ -0,0 +1,127 @@
+﻿#Requires -Modules AWSPowerShell
+
+Set-StrictMode -Version 3.0
+
+Function Usage
+{
+# startcode sqs-create-queues-endpoint-usage-powershell
+# For NServiceBus 6 Endpoints
+CreateQueuesForEndpoint -EndpointName "myendpoint" -MaxTimeToLive "2:0:0"
+
+# For NServiceBus 5 and below Endpoints
+CreateQueuesForEndpoint -EndpointName "myendpoint" -MaxTimeToLive "2:0:0" -IncludeRetries
+# endcode
+
+# startcode sqs-create-queues-shared-usage-powershell
+CreateQueue -QueueName "error" -MaxTimeToLive "2:0:0"
+CreateQueue -QueueName "audit" -MaxTimeToLive "2:0:0"
+# endcode
+}
+
+# startcode sqs-create-queues-for-endpoint-powershell
+Function CreateQueuesForEndpoint
+{
+param(
+[Parameter(Mandatory=$true)]
+[ValidateNotNullOrEmpty()]
+[string] $EndpointName,
+
+[Parameter(Mandatory=$false)]
+[ValidateNotNullOrEmpty()]
+[ValidateScript({ValidateMaxTimeToLive -MaxTimeToLive $_})]
+[string] $MaxTimeToLive = "4:0:0:0",
+
+[Parameter(HelpMessage="Only required for NSB Versions 5 and below")]
+[Switch] $IncludeRetries
+)
+
+# main queue
+CreateQueue -QueueName $EndpointName -MaxTimeToLive $MaxTimeToLive
+
+# timeout queue
+CreateQueue -QueueName "$EndpointName.timeouts" -MaxTimeToLive $MaxTimeToLive
+
+# timeout dispatcher queue
+CreateQueue -QueueName "$EndpointName.timeoutsdispatcher" -MaxTimeToLive $MaxTimeToLive

```

Comment:
Good point. No the core uses

https://github.com/Particular/NServiceBus/blob/926db772c885d729591352713809d40267d18bc3/src/NServiceBus.Core/DelayedDelivery/TimeoutManager/TimeoutManager.cs#L66 and

https://github.com/Particular/NServiceBus/blob/926db772c885d729591352713809d40267d18bc3/src/NServiceBus.Core/DelayedDelivery/TimeoutManager/TimeoutManager.cs#L86

Will change. Good catch